File: Goorm/Introduction-To-Algorithm/Chapter01/problem01D.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

int getSum(int data[], int n) {
  int answer = 0;

  for (int i = 0 ; i < n; i++) {
    answer += data[i];
  }

  return answer;
}

int main() {
  int n;
  int * data;

  scanf("%d", &n);
  data = new int[n];

  for (int i = 0 ; i < n ; i++) {
    scanf("%d", &data[i]);
  }

  int answer = getSum(data, n);
  printf("%d\n", answer);

  delete[] data;

  return 0;
}

File: Goorm/Introduction-To-Algorithm/Chapter01/problem01D.py
def get_sum(data, n):
  sum = 0
  i = 0
  while i < n:
    sum += data[i]
    i += 1
  return sum

if __name__ == '__main__':
  n = int(input())
  data = [int(word) for word in input().split()]
  answer = get_sum(data, n)
  print(answer)